src/components/warning-component.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { StyleSheet, Text, View, Image } from 'react-native';
+import colors from '@styles/colors';
+
+enum warningType {
+    Thunderstorm,
+    Strong_wind,
+    Snowstorm,
+    Rain,
+    Fog,
+    Little_wind,
+    hail,
+}
+
+export function Warning(War: warningType) {
+    switch (War) {
+        case warningType.Thunderstorm:
+            return 'Возможна гроза';
+            break;
+        case warningType.Strong_wind:
+            return 'Возможно сильный ветер';
+            break;
+        case warningType.Snowstorm:
+            return 'Возможна метель';
+            break;
+        case warningType.Rain:
+            return 'Возможно дождь';
+            break;
+        case warningType.Fog:
+            return 'Возможно туман';
+            break;
+        case warningType.Little_wind:
+            return 'Возможно слабый ветер';
+            break;
+        case warningType.hail:
+            return 'Возможно град';
+            break;
+        default:
+            return 'Предупреждений нет';
+    }
+}
+
+export default function Shape() {
+    return (
+        <View>
+            <View style={styles.appearance}>
+                <Image source={require('@assets/icon/flash_24.png')} style={styles.icon} />
+                <Text style={styles.text}>{Warning(warningType.Thunderstorm)}</Text>
+            </View>
+        </View>
+    );
+}
```

Comment:
It is better to name component name like file name. 
![image](https://user-images.githubusercontent.com/52620425/223992020-62c46b71-2dfb-4cf4-9222-92c2f17317a8.png)
Also for component names it is better to use camel case and don't write "component" in name, it is unnecessary because our function returns JSX code, already meaning it is a component.
![image](https://user-images.githubusercontent.com/52620425/223992454-348b36b0-00d9-4d4f-944b-ae3889d919cf.png)
It has no props, so how would we use this component?
As siad before better to use arrow function, so it would be
`const Warning: React.FC = props => {}`
As we are using typescript we need component type to be React.FC (meaning FunctionalComponent), also we need to have typed props so we add their type in generic of React.FC: 
`React.FC<PropsType>` 
We need warning type in props so it would be:
`...<{warningType: WarningType}>`
In the future we might need more props so it would be better to create interface describing them:
`export interface ISomethingProps {
    some1: SomeType1;
    some2: SomeType2;
}`